scripts/start-sync-server.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+SRCROOT=$(cd "$(dirname "$0")/.." && pwd)
+
+if [[ -z "$(command -v docker)" ]]; then
+  echo "starting stitch requires docker"
+  exit 1
+fi
+EXISTING_STITCH=$(docker ps | grep "mongodb-realm-test-server.*9090")
+if [[ -n "$EXISTING_STITCH" ]]; then
+  echo "found existing stitch running, not attempting to start another"
+else
+  echo "no existing stitch instance running in docker, attempting to start one"
+  . "${SRCROOT}/dependencies.list"
```

Comment:
I would like if we use the more explicit `source` command instead of the alias ".".
```suggestion
  source "${SRCROOT}/dependencies.list"
```